lang/dart.md

```diff
@@ -0,0 +1,188 @@
+---
+lang: Dart
+syntax: dart
+title: Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+
+## What is Dart?
+- [Dart](https://dart.dev) is a dynamic, class-based, object-oriented programming language with closure `and` lexical scope. Syntactically, it is quite similar to Java, C, `and` JavaScript. `If` you know any of these programming languages, you can easily learn the Dart programming language.
+
+- [Dart](https://dart.dev) is an open-source programming language that is widely used to develop the mobile application, modern web applications, desktop applications, `and` the Internet of Things (IoT) using by Flatter framework. It also supports a few advanced concepts such as interfaces, mixins, abstract classes, refield generics, `and` type  interfaces. It is a compiled language and supports two types of compilation techniques. To explore & learn, redirect to [dart tutorials](https://dart.dev/tutorials).
+
+## What is Dart used for?
+Dart programming language can be used for web and mobile applications and is also suitable for server and desktop applications.
+Dart wasn’t popular and was known as an orphan, but it had a compiler for a Dart VM, also Google had all the control about this language.
+
+**Client Web App Frameworks:**
+- **[AngularDart](https://github.com/angulardart/angular)** - AngularDart is a development platform for building mobile and desktop web applications.
+- **[Flutter](https://flutter.dev/)** - Flutter is a framework to build high-performance, cross-platform mobile apps, allowing applications to be written for Android, iOS and Web Apps.
+- **[MDL/Dart](http://mdl.mikemitterer.at)** - Material Design Lite for Dart is a framework of components for web developers based on Google's Material Design philosophy.
+- **[OverReact](https://workiva.github.io/over_react)** - A library for building statically-typed React UI components.
+- **[VueDart](https://vuedart.dev)** - Create Vue web apps using Dart, a progressive, incrementally-adoptable framework for building Web UIs.
+
+**Server Frameworks:**
+- **[Jaguar](https://github.com/Jaguar-dart/jaguar)** - A server framework built for speed, simplicity and extensibility.
+- **[Start](https://github.com/lvivski/start)** - Sinatra inspired web framework to serve static files, handle dynamic requests, websockets and create JSON responses.
+- **[Shelf](https://pub.dev/packages/shelf)** - Shelf makes it easy to create and compose web servers and parts of web servers.
+- **[Vane](https://github.com/Scorpiion/Vane)** - Framework with built-in server runtime environment and middleware system.
+- **[Rikulo Stream](https://github.com/rikulo/stream)** - Lightweight web server with request routing, filtering, template engine, WebSocket, MVC design pattern, and file-based static resources.
+- **[Alfred]( https://github.com/rknell/alfred)** - Minimal ExpressJS-inspired server with routing and middlware.
+ 
+## Why to use Dart?
```

Comment:
Same here.
```suggestion
## Why use Dart?

```

Reply:
@Panquesito7: Sorry, it's still there, likewise, I've to mark it resolved! : )

Reply:
Updated, forgot that 😅 it got updated by Grammarly.